docs/data-sources/workspace.md

```diff
@@ -16,6 +16,11 @@ Workspace data source
 data "astro_workspace" "example" {
   id = "clozc036j01to01jrlgvueo8t"
 }
+
+# Output the workspace value using terraform apply
+output "workspace" {
```

Comment:
Some say example_RESOURCE and some just say RESOURCE, we should standardize on one